src/main/java/org/jabref/gui/util/DefaultFileUpdateMonitor.java

```diff
@@ -69,10 +71,17 @@ private void notifyAboutChange(Path path) {
     @Override
     public void addListenerForFile(Path file, FileUpdateListener listener) throws IOException {
         // We can't watch files directly, so monitor their parent directory for updates
+
         Path directory = file.toAbsolutePath().getParent();
-        directory.register(watcher, StandardWatchEventKinds.ENTRY_MODIFY);
+                System.out.println(directory);
```

Comment:
please remove debug logging here and below.

I think we should go over to proper PRs also against the maintable-beta branch (except for very minor changes).

Reply:
Sorry, I will remove that later. Had no more time